xml/System.IO/Directory.xml

```diff
@@ -3476,7 +3476,12 @@ Directory::CreateDirectory("Public\\Html");
   
  [!code-csharp[System.IO.Directory#14](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.IO.Directory/CS/class6.cs#14)]
  [!code-vb[System.IO.Directory#14](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.IO.Directory/VB/class6.vb#14)]  
-  
+
+
+> [!NOTE]
+
+>  Starting on .NET Core 3.0, the <xref:System.IO.Directory.Move(System.String,System.String)> method will throw <xref:System.IO.Exception> in all platforms when the `destDirName` already exists. Before 3.0, the exception was only thrown in Windows, and other platforms could either fail or overwrite the `destDirName`. See [C++ rename](https://linux.die.net/man/2/rename).
+
  ]]></format>
         </remarks>
         <exception cref="T:System.IO.IOException">An attempt was made to move a directory to a different volume.  
```

Comment:
- The note should be placed in the Remarks section, not the Examples section.
- It would be useful to add "See the Note in the Remarks section." to line 3486.

Reply:
OK I will do that after commiting your other suggestions. Thanks.

Reply:
Done.